src/browser/disabled_frame_group_tracker.ts

```diff
@@ -228,111 +186,70 @@ function handleResizeOnly(leaderRectIndex: number, startMove: Move, end: Rectang
+                boundsChanging = true;
+                moved = new Set<OpenFinWindow>();
+                const moves = handleBoundsChanging(win, rawPayloadBounds, changeType);
+                handleBatchedMove(moves, changeType, true);
+                moves.forEach(({ofWin}) => moved.add(ofWin));
+                win.browserWindow.once('end-user-bounds-change', () => {
+                    boundsChanging = false;
+                    moved.forEach((movedWin) => {
+                        const isLeader = movedWin === win;
+                        if (!isLeader) {
+                            // bounds-changed is emitted for the leader, but not other windows
                             const endPosition = moveFromOpenFinWindow(movedWin);
-                            const isLeader = movedWin === win;
-                            promises.push(emitChange(endPosition, changeType, isLeader ? 'self' : 'group'));
-                            if (isLeader) {
-                                promises.push(raiseEvent(movedWin, 'end-user-bounds-changing', {
-                                    ...getEventBounds(endPosition.rect, endPosition.offset),
-                                    windowState: getState(win.browserWindow)
-                                }));
-                            }
-                        });
-                        await promises;
-                    } catch (error) {
-                        writeToLog('error', error);
-                    }
+                            emitChange('bounds-changed', endPosition, changeType, 'group');
+                        }
+                    });
                 });
+            } else {
+                const moves = handleBoundsChanging(win, rawPayloadBounds, changeType);
+                handleBatchedMove(moves, changeType, true);
+                moves.forEach((move) => moved.add(move.ofWin));
+                // bounds-changing is not emitted for the leader, but is for the other windows
+                const leaderMove = moves.find(({ofWin}) => ofWin.uuid === win.uuid && ofWin.name === win.name);
+                emitChange('bounds-changing', leaderMove, changeType, 'self');
             }
         } catch (error) {
             writeToLog('error', error);
         }
     };
-
-    listenerCache.set(win.browserWindow.nativeId, listener);
-    win.browserWindow.on('disabled-frame-bounds-changing', listener);
-}
-
-export function removeWindowFromGroup(win: OpenFinWindow) {
-    if (!win.browserWindow.isDestroyed()) {
-        win.browserWindow.setUserMovementEnabled(true);
-        const winId = win.browserWindow.nativeId;
-        const listener = listenerCache.get(winId);
-        if (listener) {
-            win.browserWindow.removeListener('disabled-frame-bounds-changing', listener);
+    const moveListener = (e: any, bounds: RectangleBase) => genericListener(e, bounds, 0);
+    const resizeListener = (e: any, bounds: RectangleBase) => genericListener(e, bounds, 1);
```

Comment:
Should this be changetype 2 when the origin of the window changes? That's current behavior

Reply:
Isn't that move and resize? I don't think its possible for that to happen. Will-resize and will-move events are only for user generated moves.

Reply:
No, it's change in position and size, so top-left resize is changetype 2

Reply:
I'm not concerned about the wrong changetype for our internal logic, but we should make sure we send the right info out on the event listeners

Reply:
Not seeing that behavior with bounds-changing.  Seems like a bug if that is true with other events.

Reply:
It's true with disabled-frame-bounds-changing

Reply:
spoke off-line, disabled-frame being changetype 2 is a bug